modules/virt-installing-virt-operator.adoc

```diff
@@ -26,9 +26,7 @@ You can install the {VirtProductName} Operator from the {product-title} web cons
 
 . On the *Install Operator* page:
```

Comment:
Not your change but something I noticed while looking at the preview - Step 1 of this procedure seems redundant because logging in to the console is listed as a prerequisite.

Reply:
Removed.